### DISCUSSION TITLE
How does the Radix site handle hiding scrollbar?

### DISCUSSION THREAD

**Discussion by zkSoju** [radix-ui/primitives#1230] [Help]
With most Radix primitives like (https://www.radix-ui.com/docs/primitives/components/dialog) when hiding/showing scrollbar on Windows it shifts the entire page for me. I was curious how it's fluid on the Radix site.

**Comment by zkSoju** [comment]
Notice another discussion post on this, also was this fixed in a recent update? It is not longer an issue.

**Reply by benoitgrelard** [reply]
Mostly it should work by default for anything that is positioned normally as the library we use to lock scroll fills the gap with some padding or margin.

The only remaining issue normally is for floating elements like position fixed. We have this on the docs for example with our nav. For these, there is a CSS class you can import from the lib that will deal with it.

Here's an example where we fixed a case on the docs: https://github.com/radix-ui/website/commit/b18d834135418b6c809cd3a8187418c01c2dbe2b
